src/js/app.js

```diff
@@ -0,0 +1,113 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $toggleInput = document.querySelector(`#todo-list`);
+const $todoCount = document.querySelector(`span strong`);
+
+$todoInput.addEventListener("keyup", addTodoItem);
+
+document.addEventListener("click", activeTodoItem);
+document.addEventListener("click", allTodoItem);
+document.addEventListener("click", finishTodoItem);
+
+function renderTodoItemTemplate(title) {
+    return ` <li class="active">
+                  <div class="view">
+                      <input class="toggle" onclick="onToggleTodoItem(event)" type="checkbox" >
+                      <label class="label" ondblclick="updateTodoItem(event)">${title}</label>
+                      <button class="destroy" onclick="deleteTodoItem(event)"></button>
+                  </div>
+                  <input class="edit" value=${title} />
+              </li>`;
+}
+
+function addTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
+    if ("Enter" === event.key && "" !== todoTitle) {
+        todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
+        event.target.value = "";
+        count();
+    }
+}
+
+function onToggleTodoItem(event) {
+    const target = event.target;
+    if ("toggle" === target.className) {
+        if ("completed" === target.closest("li").className) {
+            target.closest("li").className = "active";
+        } else {
+            target.closest("li").className = "completed";
+        }
+    }
+}
+
+function updateTodoItem(event) {
+    const target = event.target;
+    if ("label" === target.className) {
+        target.closest("li").classList.add("editing");
+
+        const editInput = document.querySelector(".edit");
+        editInput.addEventListener("keydown", function(event){
+            if ("Escape" === event.key) {
+                target.closest("li").classList.remove("editing");
+            }
+
+            if ("Enter" === event.key) {
+                target.innerText = editInput.value;
+                target.closest("li").classList.remove("editing");
+            }
```

Comment:
```suggestion
            if ("Escape" === event.key) {
                target.closest("li").classList.remove("editing");
                return;
            }

            if ("Enter" === event.key) {
                target.innerText = editInput.value;
                target.closest("li").classList.remove("editing");
                return;
            }
```

"Escape" 와 "Enter"가 동시에 입력되는 경우는 없겠지만, 명시적으로 `return`을 기재해주면 가독성이 더 좋을 것 같아요!